docs/build-localization-locally.md

```diff
@@ -0,0 +1,26 @@
+# Building Localization on Local Machine
+
+## Building the English File
+
+To build the latest English translation resjson file from the YAML files:
+
+* Run `npm run build` to build translations for the entire repository
+* Run `npm run build-translations` in any package directory (desktop, packages/*) to build translations for a specific package
+
+The output will be in `{packageName}/i18n/resources.resjson`
+
+## Building Translations Files for Other Languages
+
+To build the localization translations for all languages besides English:
+
+* Follow the steps above first
+* Install the latest, recommended version of nuget.exe from <https://www.nuget.org/downloads> at C:\Users\{userName}, for instance.
+* Navigate to the root of the repository
+* Run `npm run loc:restore` to install all dependencies
+* Run `npm run loc:build` to build the translations and move them to their correct directories
+* If needed, run `npm run loc:clean` to clear out all previouslly installed dependencies
```

Comment:
I don't think that we need a clean to the NuGet dependencies, but it'd be great if when people ran `npm run clean` from the repo root, it would delete the directory which contains localization build output (`Localization/out` I think)

Reply:
Alternatively, `loc:clean` could just delete `Localization/out`